XenModel/Actions/VMSS/CreateVMSS.cs

```diff
@@ -0,0 +1,75 @@
+﻿/* Copyright (c) Citrix Systems Inc. 
```

Comment:
I think the CreateVMSS action and the GetVMSSAlertsAction should not be needed: we should template CreateVMPP and GetVMPPAlertsAction instead.